po/messaging-app.pot

```diff
@@ -8,7 +8,7 @@ msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2022-01-17 08:59+0000\n"
+"POT-Creation-Date: 2022-03-29 07:40+0000\n"
```

Comment:
Seems pot file does not contain the new strings?

Reply:
true, sorry, done.